scripts/make_universal.py

```
@@ -0,0 +1,215 @@
+		if not os.path.exists(script_link):
+			os.symlink(SCRIPTS_DIR, script_link)
+
+		for item in filelist:
+			original = os.path.join(SRC_DIR, item)
+			os.symlink(original, os.path.join(self.build_dir, item))
+
+	def make(self, make_threads: int, nnue_embed_obj: str):
+		env = os.environ.copy()
+		env["ARCH"] = self.name
+		mangled = f"_ZN{len(self.namespace)}{self.namespace}4mainEiPPc"
+		if "CXXFLAGS" not in env: env["CXXFLAGS"] = ""
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)